tests/unit/test_k8s_service_name.py

```diff
@@ -0,0 +1,70 @@
+#  Copyright 2023 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import pytest
+
+from aap_eda.core.utils.k8s_service_name import (
+    InvalidRFC1035Label,
+    create_k8s_service_name,
+    rfc_1035_complicant,
+)
+
+
+def test_rfc_1035_complicant():
```

Comment:
@hsong-rh Should we use a parameterized test for all successful and failed tests, if we had an array of strings it will be much easier to see all the ones that pass and ones that fail and add new ones down the road.